### PR TITLE
Docs-related release notes

### PR DESCRIPTION
**Pull request by codyhoag**
@vikram-redhat here's a few notes/requests on the RN entries you requested earlier:
- I settled on `Documentation conventions` to list these RNs under, since they don't really fit under any other existing headings. Thoughts?
- Can you review the Apache license RN? I want to make sure I got that right.
- Lindsey approved the RN for the Copy button. We decided to leave out the `$` and `#` exclusion functionality explanation because it seems to be an expected behavior. Do you agree?
- The ARO-related release note is in progress with Kevin; I hope to have something out for that later tonight/early tomorrow.

Thanks!

### CODE REVIEW

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -453,6 +453,32 @@ installation does not fail {product-title} cluster installation.
...
+copy all text in a code block to your machine's clipboard. The Customer Portal
+will not have this capability.
+
+[id="ocp-4-4-oce-renamed-to-oke"]
+==== OpenShift Container Engine renamed to OpenShift Kubernetes Engine
+
+Red Hat has decided to rename Red Hat OpenShift Container Engine to Red Hat
+OpenShift Kubernetes Engine in order to better communicate what value the
+product offering delivers. For more information, see
+xref:../welcome/oke_about.adoc#oke-about[About OpenShift Kubernetes Engine].
```
Comment (vikram-redhat) [inline release_notes/ocp-4-4-release-notes.adoc:480]: Might need to change this xref to a fixed link as the portal doesn't have this welcome book. The link can be to the 4.4 version of this page as it is the same on all 4.x versions.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -453,6 +453,32 @@ installation does not fail {product-title} cluster installation.
...
+link:https://www.apache.org/licenses/LICENSE-2.0[Apache license 2.0]. It was
+previously licensed under the Creative Commons Attribution-ShareAlike 3.0	
+Unported license.
+
+[id="ocp-4-4-copy-button-for-docs-openshift-com-site"]
+==== Copy button for docs.openshift.com site
+
+All code blocks on docs.openshift.com now provide a Copy button that lets you
+copy all text in a code block to your machine's clipboard. The Customer Portal
+will not have this capability.
```
Comment (vikram-redhat) [inline release_notes/ocp-4-4-release-notes.adoc:472]: This capability is not available on the customer portal version of OpenShift docs.

**Comment by vikram-redhat** [comment]
2 minor nits, otherwise lgtm

**Comment by codyhoag** [comment]
Thanks, Vikram. I've made those adjustments. Merging now...

**Pull request merged**